core/lib/stack-packs.js

```diff
@@ -67,6 +67,10 @@ const stackPacksToInclude = [
     packId: 'react',
     requiredStacks: ['js:react'],
   },
+  {
```

Comment:
this should be above, with the other hosted-site creator solutions. That way they show about framework stuff

Reply:
#15039